### PR TITLE
dataset transformer 

### PR DESCRIPTION
**Pull request by Saran-nns**
Added helper function to return the transformed train and validation data loaders along with the scaler. The scaler helps to retransform the predicted trajectory later during inference.

### CODE REVIEW

File: traja/models/nn.py [hunk 1]
```diff
@@ -69,6 +68,29 @@ def get_timeseries_data_loaders(data_frame, sequence_length, train_fraction, bat
 
     return train_loader, validation_loader
 
+def get_transformed_timeseries_dataloaders(data_frame, sequence_length, train_fraction, batch_size, shift):
+
```
Comment (justinshenk) [inline traja/models/nn.py:72]: Can you please document this method, what does it do? And typo annotations for arguments and return variables would be helpful.

File: traja/models/nn.py [hunk 2]
```diff
@@ -45,8 +45,7 @@ def __getitem__(self, index):
         data = (self.data.values[index * self.sequence_length: (index + 1) * self.sequence_length],
                 self.data.values[index * self.sequence_length + self.shift: (index + 1) * self.sequence_length + self.shift])
         return data
-
-
+    
 def get_timeseries_data_loaders(data_frame, sequence_length, train_fraction, batch_size, shift):
```
Comment (justinshenk) [inline traja/models/nn.py:50]: This method could use a commend and type hints as well. :)

File: traja/models/nn.py [hunk 3]
```diff
@@ -69,6 +68,29 @@ def get_timeseries_data_loaders(data_frame, sequence_length, train_fraction, bat
 
     return train_loader, validation_loader
 
+def get_transformed_timeseries_dataloaders(data_frame, sequence_length, train_fraction, batch_size, shift):
```
Comment (WolfByttner) [inline traja/models/nn.py:71]: We don't need a function that doesn't normalise, since the user will (almost always) want to do so.

**Review by WolfByttner — CHANGES_REQUESTED** [review]

File: traja/models/nn.py (hunk 3 above)
Comment (Saran-nns) [reply traja/models/nn.py]: yes as @WolfByttner pointed out, the data has to be scaled between 0 and 1 while training RNNs. `get_timeseries_dataloaders` method is now replaced by the `get_transformed_timeseries_dataloaders`. Also, I removed the shift variable from the TimeseriesDataset class which is not required.

File: traja/models/nn.py (hunk 1 above)
Comment (Saran-nns) [reply traja/models/nn.py:62]: I committed the requested changes now. The PR is targetted towards the `upstream` branch `NeuralNetworks` and not towards `master` branch ;)

**Review by justinshenk — APPROVED** [review]
LGTM

**Review by WolfByttner — APPROVED** [review]

**Review by WolfByttner — APPROVED** [review]
